# This represents a 2-D grid map, in which each cell represents the probability of
# occupancy.

Header header 



# The map data, in row-major order, starting with (0,0).
string[] uuid_src
string[] uuid_dest
string[] name
string[] uuid
